3634/3/app/views/restaurants/show.html.slim

```diff
@@ -0,0 +1,32 @@
+.container.w-75.p-4.text-light
+  p.text-right
+    a{href="/" class="text-light"} CLOSE
+  h1.m-4.text-center = @restaurant.name
+  .border.m-4
+    p.m-3.text-justify = @restaurant.description
+    p.m-3.text-center
+      | Latitude:&nbsp;&nbsp;
+      | #{@restaurant.latitude} &nbsp;&nbsp;
+      | Longitude:&nbsp;&nbsp;
+      | #{@restaurant.longitude}
+    p.m-3.text-center
+      - if @raiting
+          | Average mark: #{@raiting.round}
+      - else
+          | Average mark: #{0}
+    .text-center
+      - if logged_in?
+        form action="/restaurants/#{@restaurant.id}/comments" method="GET"
+          button.btn.btn-success type='Submit'
+            | Leave commentary
+      - else
+        form action="/login" method="GET"
+          button.btn.btn-primary type='Submit'
+            | Login/Register
+    h3.m-3.text-center Commentary:
+    - @restaurant.comments.includes(:user).each do |comment|
```

Comment:
move comments fetching to the action. views from displaying, controller for logic